services/tables/src/main/java/com/linkedin/openhouse/tables/controller/TablesController.java

```diff
@@ -291,4 +294,39 @@ public ResponseEntity<GetAclPoliciesResponseBody> getAclPoliciesForUserPrincipal
     return new ResponseEntity<>(
         apiResponse.getResponseBody(), apiResponse.getHttpHeaders(), apiResponse.getHttpStatus());
   }
+
+  @Operation(
+      summary = "Get temporary credentials to access data in a given table",
+      description =
+          "Returns the temporary credentials which have data access to the Table resource identified by by "
+              + "databaseId and tableId. The expiration time of the temporary credentials is returned "
+              + "in millis since epoch.",
+      tags = {"Table"})
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "Access GET: OK"),
+        @ApiResponse(responseCode = "400", description = "Access GET: ACCESS_UNSUPPORTED"),
```

Comment:
HTTP status 400 should be associated with bad request such invalid input is provided. Can we consider some other HTTP code for unsupported?